### PR TITLE
Change Features Class to Service Interface

### PR DESCRIPTION
**Pull request by bosschaert**
Now that Features is a Service spec, change the status Features class to
a Service Interface

Signed-off-by: David Bosschaert <bosschae@adobe.com>

### CODE REVIEW

File: org.osgi.service.feature/src/org/osgi/service/feature/Features.java [hunk 1]
```diff
@@ -13,54 +13,41 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  *
- * SPDX-License-Identifier: Apache-2.0 
+ * SPDX-License-Identifier: Apache-2.0
  *******************************************************************************/
 package org.osgi.service.feature;
 
+import org.osgi.annotation.versioning.ProviderType;
+
```
Comment (bjhargrave) [inline org.osgi.service.feature/src/org/osgi/service/feature/Features.java:21]: Why is this moved? Are you using the Eclipse formatting configured for this project? We should not add these changes which formatting will change again. I am not saying you must use Eclipse to work on the code. But you still need to not change the code in ways that will change it again when formatted.

Comment (bosschaert) [reply org.osgi.service.feature/src/org/osgi/service/feature/Features.java]: Fixed

Comment (bjhargrave) [reply org.osgi.service.feature/src/org/osgi/service/feature/Features.java]: You should squash the commits into one commit.

**Comment by stbischof** [comment]
An option would be to rename 'Features' into 'FeatureService'

**Comment by bjhargrave** [comment]
> An option would be to rename 'Features' into 'FeatureService'

Yes, this makes a lot of sense.

**Comment by bosschaert** [comment]
> > An option would be to rename 'Features' into 'FeatureService'
> 
> Yes, this makes a lot of sense.

I've made this change now.

**Review by bjhargrave — APPROVED** [review]
